pavelib/assets.py

```diff
@@ -778,10 +779,19 @@ def webpack(options):
     result = Env.get_django_settings(['STATIC_ROOT', 'WEBPACK_CONFIG_PATH'], "lms", settings=settings)
     static_root_lms, config_path = result
     static_root_cms, = Env.get_django_settings(["STATIC_ROOT"], "cms", settings=settings)
-    environment = 'NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms}'.format(
+    js_env_extra_config_setting, = Env.get_django_settings(["JS_ENV_EXTRA_CONFIG"], "cms", settings=settings)
+    # the reason behind the following null check is because when testing, devstack settings might be null
+    if js_env_extra_config_setting is None:
+        js_env_extra_config_setting = str({})
+    js_env_extra_config = json.dumps(js_env_extra_config_setting.replace("'", '"'))
+    environment = (
+        "NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms} "
+        "JS_ENV_EXTRA_CONFIG={js_env_extra_config}"
+    ).format(
         node_env="development" if config_path == 'webpack.dev.config.js' else "production",
         static_root_lms=static_root_lms,
-        static_root_cms=static_root_cms
+        static_root_cms=static_root_cms,
+        js_env_extra_config=js_env_extra_config
```

Comment:
Nit: please add a trailing comma, to get nicer diffs in the future.

Reply:
Yeap, sorry about missing that 👍🏼 This should be addressed now 😃